rust/src/web/mod.rs

```diff
@@ -49,7 +49,7 @@ pub async fn start_web_server<A: net::ToSocketAddrs, P: AsRef<Path>>(
             .service(
                 web::resource(ENDPOINT_GRAPHQL)
                     .guard(guard::Get())
-                    .to(index_graphiql),
+                    .to(indexer_graphiql),
```

Comment:
This is fine as either `index_graphiql` or `indexer_graphiql`. I think the original intention as `index` was meaning it sits at the index of the host (e.g. `http://host/`)